kafka-connect-redis/README.md

```diff
@@ -0,0 +1,79 @@
+# Redis KCQL
```

Comment:
Are the docs inline with this? Might be best to rely on the docs alone to avoid inconsistencies.